frontend/src/lib/stores.ts

```diff
@@ -3,6 +3,9 @@ import {
 } from "svelte/store";
 import type { Stock } from "$lib/types";
 
+export const authenticated = writable(false);
+export const authenticatedUser = writable("");
```

Comment:
I would merge these into a single store, of type `export const user = Writable<{username: string} | null>`. That way you can do things like `if ($user)`.